doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -310,6 +310,20 @@ Refer to [this section](kuberay-external-storage-namespace-example) in the earli
 
 * `ENABLE_GCS_FT_REDIS_CLEANUP`: The feature gate `ENABLE_GCS_FT_REDIS_CLEANUP` is true by default, and users can turn if off by setting the environment variable in [KubeRay operator's Helm chart](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml).
```

Comment:
```suggestion
* `ENABLE_GCS_FT_REDIS_CLEANUP`: True by default. You can turn this feature off by setting the environment variable in the [KubeRay operator's Helm chart](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml).
```

Reply:
What is the difference between disabling this feature and turning it off by setting the environment variable in the Helm chart?

Reply:
Setting the environment variable in the Helm chart is the only way to disable the feature if you deploy kuberay with helm.